composeApp/build.gradle.kts

```diff
@@ -81,3 +85,43 @@ android {
         targetCompatibility(libs.versions.java.get())
     }
 }
+
+buildkonfig {
+    packageName = "com.revenuecat.purchases.kmp.sample"
+
+    defaultConfigs {
+        // apiKey is overridden in targetConfigs.
+        buildConfigField(type = STRING, name = "apiKey", value = "")
+        buildConfigField(
+            type = STRING,
+            name = "appUserId",
+            value = project.rootProject.getLocalProperty("revenuecat.appUserId").orEmpty()
+        )
+    }
+    targetConfigs {
+        create("android") {
+            buildConfigField(
+                type = STRING,
+                name = "apiKey",
+                value = project.rootProject
+                    .getLocalProperty("revenuecat.apiKey.google")
```

Comment:
We probably should add the same in purchases android 😅

Reply:
True! 😄